### ISSUE TITLE
Question about the Azure Function

### ISSUE THREAD

**Issue by frankhale** [Gnredwards/EasyAuthDemo#1]
I'd like to thank you for providing this example and also the blog post on Medium. I'm currently confused as there is no code for the Azure Function. Are you just configuring an Azure Function app with no actual code specifically for the authentication token or is this function implied in some way that the reader has to determine? 

I'm doing my best to try to understand how to authentication a client side Blazor app against Azure Active Directory and your example is the closest thing I can find. Thank you!

**Comment by frankhale** [comment]
Yeah it works with just an empty Azure Function configuration (eg. I don't actually deploy any code, just configure an empty function).

One question though, when I logout I am confronted with a failed CORS request to the logout URL. Is there any specific configuration needed for the logout URL to work? I've even enabled "Access-Control-Allow-Credentials" in the CORS blade of my Azure Function but that seems to have no effect on the outcome. Any ideas?

**Comment by Gnredwards** [comment]
Apologies for not responding sooner, my Github account isn't that busy and I didn't realise that I had an  outstanding issue.
In the example I've got up on https://amazing-curran-5dc1cc.netlify.com I have that site set up as an allowed origin in the CORS blade but I don't have the Access-Control-Allow-Credentials enabled. You also need to set up the site as an Allowed External Redirect URL in the Authentication/Authorization blade of the function app. Other than that I don't have any suggestions, I did have problems running the code locally and only managed to get it to work if I hosted it somewhere.
